extensions/notebook/package.json

```diff
@@ -43,6 +43,11 @@
           "default": false,
           "description": "%notebook.dontPromptPythonUpdate.description%"
         },
+        "notebook.jupyterServerShutdownTimeout": {
+          "type": "number",
```

Comment:
I think you should be able to set a min here, so we should not allow negative numbers (see comment below about 0)

Reply:
I couldn't find any examples of setting the min in other extension settings and tried to use "min" here but that didn't work. With the new logic, if any negative numbers are entered by accident, there will be no timeout.

Reply:
I think there is a **minimum** setting that you can use.

Reply:
Try minimum

Reply:
Ah yes that works! Thanks :)